localhost.json.SAMPLE

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Seems this file has been deleted for some reason. Can you make sure the file exists in the project?